charts/service-deployment/values.yaml

```diff
@@ -260,3 +262,44 @@ persistentVolume:
   subPath: ""
   # -- Set to "-" or "" to disable dynamic provisioning, if undefined uses the default provisioner
   # storageClass: "-"
+
+# Database Bootstrap Hook Configuration
+databaseBootstrap:
+  # -- Whether to enable the database bootstrap hook
+  enabled: false
+  # -- Script to execute in the bootstrap job
+  script: |
+    echo "Starting database bootstrap..."
+    # Your bootstrap logic here
+    echo "Bootstrap completed!"
+  # -- Image to use for bootstrap job (defaults to postgres:alpine for DB operations)
+  image: postgres:alpine
+  # -- When to run the hook (pre-install or pre-install,pre-upgrade)
+  hookPhase: "pre-install"
+  # -- Hook deletion policy (hook-succeeded, hook-failed, or before-hook-creation)
+  hookDeletePolicy: "before-hook-creation"
```

Comment:
Should the default be "before-hook-creation,hook-succeeded"? This would ensure on success it's all cleaned up. Failed runs stay in place for review.